Feature/homepage translation 15

josealeixopc opened the pull request:
Fixes #15 

I have also updated dependencies with `bundle update`. I did this because I was not able to run `bundle install` due to an yanked `mimemagic` dependency (see this StackOverflow [post](https://stackoverflow.com/questions/66919504/your-bundle-is-locked-to-mimemagic-0-3-5-but-that-version-could-not-be-found)).

The only problem I had with the updated dependencies was [this line](https://github.com/alumniei/mentorados/pull/23/files#diff-adb90cd8708ec8c3b6e57d1170b2726f85ed938ac8aaae0cd1725716fbccdf47R10). But still, I'd like to ask someone to confirm if we really need to update the dependencies.

Once PR #21 is merged, we should be able to verify if translations work without messing with the code.

ZeCarlosCoutinho commented:
I guess we need to update them given the problem with mimemagic, right?

hugopeixoto commented:
I'll see if I can get #21 merged today and then I'll see what's up with that weird line. And yeah, we need the update. We should be keeping this up to date anyway, so it would be done sooner or later.

hugopeixoto commented:
I've merged #21. Can you rebase this PR on top of the latest main?

josealeixopc commented:
Hello! I squashed the commits and rebased. I verified that changing the settings also changes the language of the homepage, but please confirm before merging.

hugopeixoto commented on app/views/home/index.pt.html.erb:
```diff
@@ -0,0 +1,43 @@
+</p>
+
+<p>
+  Esta plataforma foi construída para juntar estudantes do MIEIC que estejam à procura de conselhos a almuni do MIEIC que voluntariam o seu tempo para ouvir e aconselhar quando podem.
```
```suggestion
  Esta plataforma foi construída para juntar estudantes do MIEIC à procura de apoio e antigues alunes que voluntariam o seu tempo para ouvir e aconselhar onde puderem.
```

hugopeixoto commented on app/views/home/index.pt.html.erb:
```diff
@@ -0,0 +1,43 @@
+<p>
+  Estudantes do MIEIC que estejam com dificuldade em decisões de carreira, que tenham problemas pessoais/profissionais na universidade, ou com quaisquer outros problemas, podem beneficiar de falar com alguém que já tenha tido uma experiência semelhante.
```
```suggestion
  Estudantes do MIEIC que estejam com dificuldade em decisões de carreira, que tenham problemas pessoais/profissionais na universidade, ou quaisquer outros problemas, podem beneficiar de falar com alguém que já tenha tido uma experiência semelhante.
```

hugopeixoto commented on app/views/home/index.pt.html.erb:
```diff
@@ -0,0 +1,43 @@
+<h2>Encontra um mentor</h2>
+
+<p>
+  Terás acesso às informações de contacto de vários mentores, assim como um resumo das suas expierências e áreas de especialidade.
```
```suggestion
  Terás acesso às informações de contacto de váries mentories, assim como um resumo das suas experiências e áreas de especialidade.
```

hugopeixoto commented on app/views/home/index.pt.html.erb:
```diff
@@ -0,0 +1,43 @@
+</p>
+
+<p>
+  Nós não te iremos pedir nenhuma informação acerca de ti, a não ser o teu endereço de email institucional.
```
```suggestion
  Não te pediremos nenhuma informação pessoal além do teu endereço de email institucional.
```

hugopeixoto commented on app/views/home/index.pt.html.erb:
```diff
@@ -0,0 +1,43 @@
+  Esta plataforma foi construída para juntar estudantes do MIEIC que estejam à procura de conselhos a almuni do MIEIC que voluntariam o seu tempo para ouvir e aconselhar quando podem.
+</p>
+
+<h2>Encontra um mentor</h2>
```
```suggestion
<h2>Encontra alguém que te apoie</h2>
```

hugopeixoto reviewed (commented):
The bundle update seems to be working. I was reading through the suggested translation and found some typos. I fixed those, and also tried to use gender neutral language (using the "e" system from https://orgulhogay.pt/neutralidade-de-genero-na-lingua-portuguesa/). I think we should make some effort to try and make the language of the platform more inclusive.

josealeixopc commented:
Done! I'm merging.
